Cargo.toml

```diff
@@ -13,11 +13,12 @@ categories = ["embedded", "hardware-support", "no-std"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cortex-m = "0.7.5"
+cortex-m = { version = "0.7.5", features = ["critical-section-single-core"] }
 embedded-hal = "0.2.7"
 switch-hal = "0.4.0"
 stm32f4xx-hal = { version = "0.13.2", features = ["stm32f446"]}
 unwrap-infallible = "0.1.5"
+rtt-target = "0.5.0"
```

Comment:
Is this needed for cargo embed for the example you added? If so, I think it's best to place it in the dev-dependencies as someone using the crate as a library has no need to add this as a dependency.

Reply:
Yes, I'm going to remove the new examples so this will be removed.